src/assets/css/login.css

```diff
@@ -0,0 +1,81 @@
+
+body {
+    background: #1A203E;
+}
+
+.textinput1 {
+    background: #0D1333;
+    border-radius: 5px;
+    border-color: #E5E5E5;
+    position: absolute;
+    right: 0px;
+    margin-right: 100px;
+    margin-top: 30px;
+    height: 30px;
+    width: 35em;
+    text-align: left;
+    color: gray;
+    font-family: Avenir;
+    font-size: 13pt;
+}
+.textinput2 {
+    background: #0D1333;
+    border-radius: 5px;
+    border-color: #E5E5E5;
+    position: absolute;
+    right: 0px;
+    margin-right: 100px;
+    margin-top: 70px;
+    height: 30px;
+    width: 35em;
+    text-align: left;
+    color: gray;
+    font-family: Avenir;
+    font-size: 13pt;
+}
```

Comment:
die beiden text inputs scheinen genau gleich zu funktionieren. lösch doch vielleicht eine und geb dann beiden boxen die gleiche klasse? klassen darfst du ja mehrfach verwenden, nur IDs nicht.

Reply:
außerdem könnten die textboxen noch eine hellere "color"-property vertragen, die schrift ist schwer zu lesen wenn man etwas eintippt.
etwas padding würde dafür sorgen, dass der text nicht so an den borders klebt.

Reply:
padding kann ich noch hinzufügen. ich hatte die beiden am anfang in einer klasse, musste die aber als floats deklarieren, weshalb ich sie dann einzeln platzieren musste, damit sie nicht aufeinander klebten

Reply:
du kannst ja alles bis auf den float in eine klasse packen, und dann die floats entweder direkt am element, oder mit einer eigenen mini-klasse jeweils zuordnen. so kannst du redundante zeilen entfernen.